scss/titles.scss

```diff
@@ -1,5 +1,17 @@
 .title {
-  font-family: "Roboto", sans-serif;
+  font-family: 'Roboto', sans-serif;
   font-size: 24px;
-  margin: 5% 7%;
+  position: relative;
+  padding: 38px 0 0 52px;
```

Comment:
remember we use margin to separate from other content, i think here you could do a better use of the position of the element ::before setting the top and left and not changing here the padding, better is to use the margin here to control both elements separations of outside elements.